L1Trigger/Phase2L1ParticleFlow/python/l1pfJetMet_cff.py

```diff
@@ -0,0 +1,38 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoMET.METProducers.PFMET_cfi import pfMet as _pfMet
+_pfMet.calculateSignificance = False
+l1PFMetCalo    = _pfMet.clone(src = "l1pfCandidates:Calo")
+l1PFMetPF      = _pfMet.clone(src = "l1pfCandidates:PF")
+l1PFMetPuppi   = _pfMet.clone(src = "l1pfCandidates:Puppi")
+
+l1PFMets = cms.Sequence(l1PFMetCalo + l1PFMetPF + l1PFMetPuppi)
```

Comment:
there should be corresponding tasks

Reply:
is that really needed any more, with the changes to `cms.Path` from CMSSW11.1.x ?

Reply:
Modules in a Task result in a little bit less overhead than modules in Path+Sequence. Task also allows framework to avoid running (and "soon" also deleting) modules whose products are not consumed.

Reply:
I'd leave this to @rekovic (maybe they can just be switched all to cms.Task, unless the Sequences are needed elsewhere)